locales/en.yml

```diff
@@ -37,7 +37,8 @@ en:
           email: "You are identified with JIRA as %{email}"
         issue:
           created: "Issue %{key} created"
-          details: "%{key}: %{summary}, assigned to: %{assigned}, priority: %{priority}, status: %{status}"
+          details: "[%{key}] %{summary}\nStatus: %{status}, assigned to: %{assigned}, fixVersion: %{fixVersion}, priority: %{priority}\n%{url}"
```

Comment:
:+1:, though I suspect a future enhancement will probably be to have variable fields (I'm sure different JIRA issues will have fixVersion, or Priority, etc).